apps/documents/templates/meinberlin_documents/document_detail.html

```diff
@@ -8,24 +8,25 @@
 
         {% for paragraph in object.paragraphs.all %}
 
-        <section class="paragraph">
-            <h2 class="paragraph-title">
-                {{ paragraph.name }}
-            </h2>
-            <div class="paragraph-comments">
+        <section class="commenting">
+            <div class="commenting-content">
+                <h2 class="commenting-title">
+                    {{ paragraph.name }}
+                </h2>
+                <div class="paragraph-body">
```

Comment:
> Should be commenting-body? But commenting-body is also not defined.

I'll remove it, as not needed.